src/dx/formatters/main.py

```diff
@@ -211,7 +212,18 @@ def format_output(
         **dataframe_info,
     )
 
-    payload = {settings.MEDIA_TYPE: payload}
+    # add additional payload for LLM consumption; if any parsing/summarizing errors occur, we
+    # shouldn't block displaying the bundle
+    llm_payload = ""
+    try:
+        llm_payload = summarize_dataframe(df)
+    except Exception as e:
+        logger.debug(f"Error in summarize_dataframe: {e}")
+
+    payload = {
+        settings.MEDIA_TYPE: payload,
+        "text/llm+plain": llm_payload,
```

Comment:
If the `summarize` errors, we should omit the `llm_payload`. Alternatively, we could include the error message in the llm_payload so that we can discover any issues.

Reply:
Sounds good, probably safer to omit for now